torch/export/_unlift.py

```diff
@@ -298,10 +301,21 @@ def _create_stateful_graph_module(
         delattr(submod, field)
         _assign_attr(buffer, stateful_gm, constant_fqn, attr_kind=_AttrKind.CONSTANT)
 
+    # Constants are not preserved well when we create a new GraphModule unlike param/buffers
+    for const_name, value in ep.constants.items():
+        if not torch.fx.graph_module._has_attr(stateful_gm, const_name):
```

Comment:
I think `_recursive_getattr` does the same thing?

Reply:
Yeah i will delete that and use the one in fx.graph_module. I think we have many duplicate implementation of a same thing haha.